### DISCUSSION TITLE
숫자 사이에 공백이 생깁니다.

### DISCUSSION THREAD

**Discussion by taeryong1324** [orioncactus/pretendard#92] [Q&A]
안녕하세요!

폰트 사용중에 숫자 1,2 뒤에 스페이스를 넣은 것처럼 간격이 뜨는 문제가 있는데요, 
다른 숫자는 그러지 않는데 1,2 뒤에 숫자가 붙을 경우 스페이스를 친 것 같은 공백이 생깁니다

다른 폰트로 변경하면 이런 문제가 사라져서 질문 드립니다
Linux, python, PIL 사용중입니다

**Comment by orioncactus** [comment]
안녕하세요! 메일로도 문의를 남겨주셔서 답장을 보내드렸습니다. 답장하시기 편한 수단으로 알려주시면 마저 확인해보겠습니다.
